Makefile.dep

```diff
@@ -224,3 +224,7 @@ endif
 ifneq (,$(filter ng_at86rf2%,$(USEMODULE)))
   USEMODULE += ng_at86rf2xx
 endif
+
+ifneq (,$(filter kw2xrf,$(USEMODULE)))
+  USEMODULE += kw2xrf
```

Comment:
you mean, because this is already added in the auto-init mechanism? But the atmel-driver also includes this here.